app/src/main/java/rick_and_morty/ui/theme/Theme.kt

```diff
@@ -55,10 +42,18 @@ fun RickandmortyTheme(
         darkTheme -> DarkColorScheme
         else -> LightColorScheme
     }
+
     val view = LocalView.current
+    val context = LocalContext.current
+    val window = when (context) {
+        is Activity -> context.window
+        is ContextWrapper -> (context.baseContext as? Activity)?.window
+        else -> null
+    }
```

Comment:
Could you elaborate on why this is needed?

Reply:
Actually I dont need this, but left it as it was standard created